.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  source-archive:
+    runs-on: ubuntu-latest
+    env:
+      FORMAT: zip
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Get version number
+      run: |
+        VERSION=$(git describe --tags --abbrev=0 --match 'v*' | tr -d '[:alpha:]')
+        [ -z "$VERSION" ] && exit 1
+        echo "VERSION=${VERSION}" >> $GITHUB_ENV
+
+    - name: Create archive
+      run: |
+        git archive --prefix ${{ env.PREFIX }} --format ${{ env.FORMAT }} HEAD > ${{ env.OUTPUT }}
```

Comment:
Shouldn't it create an archive of `vVERSION` tag?

Reply:
We only use the v in the tag name, but not for the actual version number in the release artifacts.